web/client/api/__tests__/WMS-test.js

```diff
@@ -175,6 +174,18 @@ describe('Test correctness of the WMS APIs', () => {
             }
         });
     });
+
+    it('GetRecords transform SRS List to uppercase', (done) => {
+        API.getRecords('base/web/client/test-resources/wms/GetCapabilities-1.3.1.xml', 0, 2, '').then((result) => {
```

Comment:
1.3.0 is the version of the GetCapabilities response. Here we are testing a GetCapabilities 1.3.0 that uses epsg value lowercase so please rename this file `GetCapabilities-1.3.0-epsg-lowercase.xml`.